inbm-lib/inbm_common_lib/utility.py

```diff
@@ -124,7 +124,7 @@ def remove_file(path: Union[str, Path]) -> bool:
         return False
 
 
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Refactoring$</h2>**
**Suggestion:** Use os.path.isfile instead of os.path.exists and os.path.isfile to simplify the check for whether the path is a file.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        if not os.path.isfile(canonical_path):
            return False
```